_content/libraries.md

```diff
@@ -1,26 +1,57 @@
-## SPSP
+## Connectors
 
-Build Interledger payments into your apps with the [ILP SPSP Client][1]. This library can connect to any SPSP receiver for account discovery and payment detail negotation. For more information on how to use SPSP, read ["SPSP: Simple Payment Setup Protocol"][2] on the Interledger Blog. 
+### Java Connector
+An Interledger [Connector implemented in Java][1]. This project is actively maintained by [Xpring](https://xpring.io) 
+and recommended for new production deployments. 
 
-## Koa ILP
+### Rust Connector
+An Interledger [Connector implemented in Rust][2]. This project is not actively maintained, but has a robust feature-set
+and is easy to use.
 
-If you use the [Koa framework][3] in Node.js, this library will allow you to monetize your API in only a few lines of code. For more information, [check out our Github][4] and [our blog post about it.][5]
+### Javascript Connector
+An Interledger [Connector implemented in Javascript][3]. This project is not actively maintained, but has been 
+battle-tested in various production deployments.
 
-## ILP Fetch
+## Libraries
 
-You can use [ILP Fetch][6] as a client to HTTP-ILP. ILP Fetch follows the [Fetch API][7], but allows the client to send payments. You can use it to interact with paid APIs on Interledger.
+### Interledger Java
+Build ILP applications natively in Java using [Quilt][4], which is a Java implementation of all Interledger primitives.
 
-## Moneyd XRP
+### Interledger RS
+Build ILP applications natively in Rust using [interledger.rs][5], which is a Rust implementation of all Interledger primitives.
 
-[Moneyd-XRP][8] allows you to connect to the Interledger network via XRP. It can be used on the livenet or the testnet with minimal configuration. We also have "Moneyd-ETH" and "Moneyd-Lightning" in the works.
+### Interledger JS
```

Comment:
I straight-up don't think we should direct anyone here.

I'm a new dev who gets directed to Interledger.js. This is what I see:

- Pinned repos: `ilp-connector` (fine, but not great...), `moneyd` (not good -- see other comment), `tf-connector` (hasn't been updated in 2 years, probably doesn't work)
- Then, the user sees the most "recently updated" repos, most of which are actually _not_ maintained but constantly barraged by bots to update deps. (I'm looking at you, `moneyd`, `moneyd-xrp`, `btp-packet`, `wallet`, `ilp-plugin-ethereum`, and the list goes on and on...). It's hard to find the monorepo, or ilp-protocol-stream buried in all of that. Currently (but hopefully we can fix this!) stream is probably one of the only useful modules, but even that is probably too complex to reasonably use (at least there are some examples).

How do we reasonably expect anyone to make sense of all of this!? What do we reasonably expect them to build!?

Reply:
Agreed. These are removed.

Reply:
Well, there _is_ something here, especially if you're a JS developer. Maybe we just update hte description to warn about this? What about:

"Build ILP applications natively in Javascript using InterledgerJS , which is a Javascript implementation of Interledger (note that this project has many older libraries that are no longer applicable to Interledger)."

Reply:
What if we just included direct links to these modules, rather than the interledgerjs org?

- `ilp-connector`
- `ilp-plugin-btp`
- `ilp-plugin-http`
- `ilp-protocol-stream`

Everything else is either (a) not useful, (b) doesn't work, (c) not documented well enough or too domain-specific to reasonably use without hand-holding.

Hopefully we can improve this over time by moving modules into the monorepo & deprecating old ones, documenting everything well there, and then directing everyone to the monorepo. Also, more generally, we need a better application-layer JS SDK -- which I think goes hand-in-hand with Open Payments/delegated access work.

Reply:
Agree. Made changes -- take another look?